sympy/polys/tests/test_densetools.py

```diff
@@ -613,6 +613,9 @@ def test_dup_sign_variations():
 
 
 def test_dup_clear_denoms():
+
+    x = symbols('x')
```

Comment:
This symbol is already imported at the top of the file so it does not need to be defined here.